tests/integration/utils.py

```diff
@@ -196,6 +196,40 @@ def make_list_of_packages_hashable(data):
     return sorted([[i["name"], i["type"], i["version"]] for i in data])
 
 
+def sort_cachito_response(orig_response):
+    """
+    Sorts lists of packages and dependencies for testing purpose in Cachito request.
+
+    Sorting parts:
+    * ["dependencies"] by ["name"] + ["version"]
+    * ["packages"] by ["name"] + ["version"]
+    * ["packages"][idx]["dependencies"] by ["name"] + ["version"]
+
+    :param dict orig_response: the Cachito response for sorting
+    :return dict sorted_response: sorted Cachito response
+    """
+    sorted_response = {}
+    for k, v in orig_response.items():
+        if k == "dependencies":
+            sorted_response["dependencies"] = sorted(
+                v, key=lambda dep: dep["name"] + dep["version"]
+            )
+        elif k == "packages":
+            sorted_response["packages"] = []
+            for package in v:
+                pkg_with_sorted_dep = package
+                pkg_with_sorted_dep["dependencies"] = sorted(
+                    package["dependencies"], key=lambda dep: dep["name"] + dep["version"]
+                )
+                sorted_response["packages"].append(pkg_with_sorted_dep)
+            sorted_response["packages"] = sorted(
+                sorted_response["packages"], key=lambda pkg: pkg["name"] + pkg["version"]
+            )
```

Comment:
Agree with Eliska. Additionally, I think you should consider sorting packages not just by `"name", "version"` but by `"dev", "name", "type", "version"` (the unique constraint in the database). And instead of concatenating the values into 1 string, you could use a tuple as the sorting key: `(pkg["name"], pkg["version"], pkg["type"], pkg["dev"])`

Reply:
@chmeliik In case we will use all parameters it will be different for dependencies and packages (there are dev and replaces parameters for dependencies). I would like to use the only name, version, and type for sorting.

Reply:
@akhmanova ah true, yeah ignoring dev would probably be fine (or you could just use `pkg.get("dev", False)` instead)